lib/wallaby/query.ex

```diff
@@ -29,6 +29,7 @@ defmodule Wallaby.Query do
     * `:count` - The number of elements that should be found (default: 1).
     * `:visible` - Determines if the query should return only visible elements (default: true).
     * `:text` - Text that should be found inside the element (default: nil).
+    * `:get` - The number of the element to return if multimple elements satisfy the selection criteria.
```

Comment:
Out of all of these options I think `item` or `pick` are the ones that make sense to me. I've also thought that it could just be called `nth` since thats how it works in css. Not sure if thats  the most descriptive but it would at least be familiar to folks familiar with css and it describes what you're going to do fairly well.